mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/VeilederflateService.kt

```diff
@@ -129,11 +129,11 @@ class VeilederflateService(
 
     suspend fun hentTiltaksgjennomforing(
         id: String,
-        fnr: String,
+        fnr: String?,
         accessToken: String,
     ): List<VeilederflateTiltaksgjennomforing> {
-        val brukerData = brukerService.hentBrukerdata(fnr, accessToken)
-        val enhetsId = brukerData.geografiskEnhet?.enhetsnummer ?: ""
+        val brukerData = fnr?.let { brukerService.hentBrukerdata(fnr, accessToken) }
+        val enhetsId = brukerData?.geografiskEnhet?.enhetsnummer ?: ""
```

Comment:
Hva skjer i preview-modus når `val enhetsId = ""`? Blir alle kontaktpersoner vist, eller ingen?

Reply:
Man får ingen kontaktpersoner

Reply:
Jeg kan evt. legge inn en melding i frontend for kontaktpersoner, at dersom man er i preview-modus så står det bare at kontaktpersoner ikke kan vises fordi vi ikke vet hvilken bruker man har i kontekst under preview